Realm/RLMAccessor.hpp

```diff
@@ -40,19 +41,8 @@ struct RLMOptionalId {
     id operator*() const noexcept { return value; }
 };
 
-class RLMAccessorContext {
-public:
-    ~RLMAccessorContext();
-
-    // Accessor context interface
-    RLMAccessorContext(RLMAccessorContext& parent, realm::Obj const& parent_obj, realm::Property const& property);
-
-    id box(realm::object_store::Dictionary&&);
-    id box(realm::object_store::Set&&);
-    id box(realm::List&&);
-    id box(realm::Results&&);
-    id box(realm::Object&&);
-    id box(realm::Obj&&);
+// The subset of RLMAccessorContext which does not require any member variables
```

Comment:
Comments like this really need a "why". The "what" is fairly useless and the name of the struct tells me nothing.